inst/css/BS4/pkgdown-variables.scss

```diff
@@ -0,0 +1,26 @@
+// This Sass file wants to be compiled with Bootstrap Sass headers
+// All custom properties are prefixed with `rs-` (abbr: Rmd style) to avoid conflicts
+
+// navbar links
+$rs-nav-dark-link-active-color: color-contrast($component-active-bg) !default;
+$rs-nav-dark-link-active-bg: $component-active-bg !default;
+$rs-nav-light-link-active-color: $component-active-color !default;
+$rs-nav-light-link-active-bg: $component-active-bg !default;
+$rs-nav-link-active-border-radius: $border-radius !default;
+$rs-dropdown-link-active-color: $dropdown-link-hover-color !default;
+$rs-dropdown-link-active-bg: $dropdown-link-hover-bg !default;
+
+// toc
+$rs-toc-border-radius: $border-radius !default;
+$rs-toc-border-width: 0 !default;
+$rs-toc-border-color: $border-color !default;
+$rs-toc-active-bg: $list-group-active-bg !default;
```

Comment:
@apreshill In the docs we could mention these variables that are more intuitive than `$list-group-active-bg` for instance.